bundles/admin/admin-layereditor/view/AdminLayerFormService.js

```diff
@@ -333,6 +394,6 @@ export class AdminLayerFormService {
     }
 
     notify () {
-        this.listeners.forEach(consumer => consumer());
+        this.listeners.forEach(consumer => consumer(this.getLayer()));
```

Comment:
The layer doesn't seem to be used. It's accessed from the getter instead.

Reply:
True, though I'd much rather get the modified result as param as a consumer/listener but since we don't use it now I'll remove it.